src/main/webapp/app/shared/user-settings/user-settings.route.ts

```diff
@@ -50,6 +51,13 @@ export const userSettingsState: Routes = [
                     pageTitle: 'artemisApp.userSettings.categories.SSH_SETTINGS',
                 },
             },
+            {
+                path: 'idePreferences',
+                component: IdeSettingsComponent,
+                data: {
+                    pageTitle: 'artemisApp.userSettings.categories.idePreferences',
```

Comment:
Maybe I'm blind, but I think the translation value for this is still missing. At least I can't find it in the `userSettings.json`

Reply:
<img width="854" alt="image" src="https://github.com/user-attachments/assets/23018729-82fe-4965-a354-ca872faca6cc">
should be this one right?

Reply:
yes youre right. thanks